create-ava-rule.js

```diff
@@ -129,16 +130,29 @@ module.exports = function createAvaRule() {
 		}
 	};
 
-	Object.defineProperty(rule, 'isTestFile', {
-		get: function () {
-			return isTestFile;
-		}
-	});
+	rule.isInTestFile = function () {
+		return isTestFile;
+	};
 
-	Object.defineProperty(rule, 'currentTestNode', {
-		get: function () {
-			return currentTestNode;
-		}
+	rule.isTestNode = function (node) {
+		return currentTestNode === node;
+	};
+
+	rule.isInTestNode = function () {
+		return currentTestNode;
+	};
+
+	rule.if = rest(function (predicates) {
```

Comment:
In Atom for instance, the `return` method of Bluebird
```js
return Promise.resolve(foo)
  .return(bar);
```

also gets highlighted like the `return` keyword. Which is nice IMO :)